packages/lisk-api-client/package.json

```diff
@@ -19,22 +19,46 @@
 		"node": ">=6.3 <=9.5",
 		"npm": ">=3 <=5"
 	},
-	"main": "dist/index.js",
+	"browser": "dist-browser/index.min.js",
+	"main": "dist-node/index.js",
 	"scripts": {
+		"babel": "babel src -d dist-node",
+		"babel:browsertest":
+			"babel src -d ./browsertest/src && BABEL_ENV=browsertest npx babel test --ignore test/transactions/dapp.js -d ./browsertest/test",
+		"browserify":
+			"browserify ./dist-node/index.js -o ./dist-browser/index.js -s lisk --node",
+		"browserify:browsertest":
+			"browserify ./browsertest/test/*.js ./browsertest/test/**/*.js -o ./browsertest/browsertest.js -s lisk",
+		"uglify":
+			"uglifyjs -nm -o ./dist-browser/index.min.js ./dist-browser/index.js",
+		"uglify:browsertest":
+			"uglifyjs -o ./browsertest/browsertest.min.js ./browsertest/browsertest.js",
 		"format": "prettier --write \"*.{js,json}\" \"{src,test}/**/*.{js,json}\"",
 		"lint": "eslint .",
 		"lint:fix": "npm run lint -- --fix",
 		"test": "NODE_ENV=test nyc mocha test",
 		"test:watch": "npm test -- --watch",
 		"test:watch:min": "npm run test:watch -- --reporter=min",
+		"test:node":
+			"npm run build:check && BABEL_ENV=buildcheck mocha test --recursive",
+		"test:browser": "cypress run --env ROOT_DIR=\"${PWD##*/}\"",
+		"serve:browsertest": "http-server browsertest",
 		"cover":
 			"if [ -z $JENKINS_HOME ]; then npm run cover:local; else npm run cover:ci; fi",
 		"cover:base": "NODE_ENV=test nyc report",
 		"cover:local": "npm run cover:base -- --reporter=html --reporter=text",
 		"cover:ci": "npm run cover:base -- --reporter=text-lcov | coveralls -v",
-		"prebuild": "rm -r dist || mkdir dist | echo",
-		"build": "babel src -d dist",
-		"build:check": "node -e \"require('./dist')\"",
+		"build:browsertest":
+			"npm run babel:browsertest && npm run browserify:browsertest && npm run uglify:browsertest",
+		"postbuild:browsertest": "rm -r browsertest/src browsertest/test",
+		"prebuild:node": "rm -r dist-node/* || mkdir dist-node | echo",
+		"build:node": "npm run babel",
+		"prebuild:browser":
+			"rm ./dist-browser/index.js ./dist-browser/index.min.js | echo",
+		"build:browser": "npm run babel && npm run browserify && npm run uglify",
+		"prebuild": "npm run prebuild:node && npm run prebuild:browser",
+		"build": "npm run babel && npm run browserify && npm run uglify",
```

Comment:
Let's make this `npm run build:node && npm run build:browser`. Then we can remove the `npm run babel` from `build:browser`.

Reply:
@willclarktech  I did it, however now `build:browser` will fail if `build:node` wasn't run before which feels a bit wrong. I would expect `build:browser` to build browser bundle without depending on another command.